prs/PizzaStore.ts

```diff
@@ -0,0 +1,40 @@
+class PizzaStore {
+    public createPizza(style: string, type: string): Pizza | null {
```

Comment:
Looks like you are using this method as a sort of factory method.  If so I would consider making this either a `static` method on the `PizzaStore` or even consider splitting out the factory method into it's own `PizzaFactory` class.